Adding test to quantized copy for 'from float'

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#43681 Adding test to quantized copy for 'from float'**

Differential Revision: [D23364507](https://our.internmc.facebook.com/intern/diff/D23364507)

Commit:
Adding test to quantized copy for 'from float'

[ghstack-poisoned]

Commit:
Update on "Adding test to quantized copy for 'from float'"


Differential Revision: [D23364507](https://our.internmc.facebook.com/intern/diff/D23364507)

[ghstack-poisoned]

supriyar commented on test/quantization/test_quantized_tensor.py:
```diff
@@ -447,21 +447,31 @@ def test_qtensor_copy(self):
+            r = torch.randn([numel], dtype=torch.float).to(device)
+            q = torch._empty_affine_quantized([numel], scale=scale, zero_point=zero_point, dtype=dtype, device=device)
+            print("Device:", device)
+            print("Device (in q):", q.device)
```
remove these print statements?

Commit:
Update on "Adding test to quantized copy for 'from float'"


Differential Revision: [D23364507](https://our.internmc.facebook.com/intern/diff/D23364507)

[ghstack-poisoned]
